core/src/main/java/org/bitcoinj/evolution/AssetLockTransaction.java

```diff
@@ -142,14 +141,7 @@ public TransactionOutPoint getLockedOutpoint() {
         if(lockedOutpoint != null)
             return lockedOutpoint;
 
-        for(int i = 0; i < getOutputs().size(); ++i) {
-            Script script = getOutput(i).getScriptPubKey();
-            if(ScriptPattern.isCreditBurn(script)) {
-                // The lockedOutpoint must be in little endian to match Platform
-                // having a reversed txid will not allow it to be searched or matched.
-                lockedOutpoint = new TransactionOutPoint(params, i, Sha256Hash.wrap(getTxId().getReversedBytes()));
-            }
-        }
+        lockedOutpoint = new TransactionOutPoint(params, 0, Sha256Hash.wrap(getTxId().getReversedBytes()));
```

Comment:
Here we use index `0` because this AssetLockTransaction only supports a single output in the payload.

However, an assetlock transaction could have several outputs in the payload.